pkg/controller/che/che_controller.go

```diff
@@ -720,32 +729,32 @@ func (r *ReconcileChe) Reconcile(request reconcile.Request) (reconcile.Result, e
 		cheImageRepo = util.GetValue(instance.Spec.Server.CheImage, deploy.DefaultCodeReadyServerImageRepo)
 		cheImageTag = util.GetValue(instance.Spec.Server.CheImageTag, deploy.DefaultCodeReadyServerImageTag)
 	}
-	cheDeployment, err := deploy.NewCheDeployment(instance, cheImageRepo, cheImageTag, cmResourceVersion, isOpenShift)
+	cheDeploymentToCreate, err := deploy.NewCheDeployment(instance, cheImageRepo, cheImageTag, cmResourceVersion, isOpenShift)
```

Comment:
you have variables called `newKeycloakDeployment` and `cheDeploymentToCreate` ... assuming these are both new things to be created, should the names be more consistent? eg., both prefix with `new` or suffix with `ToCreate` ?

Also any chance we can call it "IdentityProvider" instead of "Keycloak" ? :) Or is that a bad refactoring to do piecemeal, which we should tackle in Che 8.0 ?

Reply:
I already tried to clean naming to make the code easier to read / understand in the most critical parts. But I might have forgotten some of left some inconsistencies. Is it a blocker for you ?

Reply:
As for the `Keycloak` to `ìdentityProvider` change, I'd prefer doing that in a next PR, probably in the context of the wider refactoring that will certainly be required after GA.

Reply:
not a blocker. +1 for wider refactoring in another PR.